src/page_components/listing/ListingCard.tsx

```diff
@@ -112,18 +118,32 @@ const ListingCard = (props: ListingCardProps) => {
   const getContentHeader = () => {
     return (
       <div className="listings-row_headers">
-        {getHeader(contentProps?.contentHeader, 2, "largePrimary", "order-1")}
-        {getHeader(contentProps?.contentSubheader, 3, "mediumNormal", "order-2")}
+        {contentProps?.contentHeader &&
+          getHeader(
+            contentProps?.contentHeader,
+            contentProps?.contentHeader?.priority ?? 2,
+            "largePrimary",
+            "order-1"
+          )}
+        {contentProps?.contentSubheader && (
+          <p className="card-subheader order-2">{contentProps?.contentSubheader?.content}</p>
+        )}
+
         {cardTags && cardTags?.length > 0 && (
           <div className="listings-row_tags">
             {cardTags?.map((cardTag, index) => {
               return (
-                <Tag styleType={cardTag.styleType || AppearanceStyleType.warning} key={index}>
+                <Tag
+                  styleType={cardTag.styleType ?? AppearanceStyleType.warning}
+                  shade={cardTag?.shadeType}
+                  key={index}
+                >
                   {cardTag.iconType && (
                     <Icon
                       size={"medium"}
                       symbol={cardTag.iconType}
                       fill={cardTag.iconColor ?? IconFillColors.primary}
+                      className={"me-2"}
```

Comment:
What does "me-2" do? It's not a margin class I recognize from tailwind.

Reply:
Oh I should remove that bc it's tailwind, it's a RTL version of the margin class, I will --> to the scss file and use the new "rtl" selector